src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -78,6 +78,8 @@
     public static final String INVALID_TAG = "ABCDE";
     public static final String INVALID_ZERO_INDEX = "0";
     public static final String INVALID_INDEX = "-69";
+    public static final String INVALID_INTEGER_MIN = Long.toString(Integer.MIN_VALUE + 1);
+    public static final String INVALID_INTEGER_MAX = Long.toString(Integer.MAX_VALUE + 1);
 
     public static final String PREAMBLE_WHITESPACE = "\t  \r  \n";
```

Comment:
Is this used? I think it wasn't used in my version so I was wondering if it should be deleted.

Reply:
Same with `INVALID_REMARK_DESC` (it's not used) but you may want to add testcases that incorporate it's use instead of deleting it! 🧼